expression/builtin_json_vec_test.go

```diff
@@ -38,10 +38,12 @@ var vecBuiltinJSONCases = map[string][]vecExprBenchCase{
 	ast.JSONSearch:       {},
 	ast.JSONReplace:      {},
 	ast.JSONDepth:        {{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETJson}}},
-	ast.JSONUnquote:      {},
-	ast.JSONRemove:       {},
-	ast.JSONMerge:        {},
-	ast.JSONInsert:       {},
+	ast.JSONUnquote: {
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETJson}},
```

Comment:
The string generated by default generator will never be a JSON, consider using `jsonStringGener` in bench_test.go.

Reply:
yes, I have used `JsonStringGener` in test case now.